src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Flows/AcquireTokenByDeviceCodeHandler.cs

```diff
@@ -34,24 +35,31 @@ namespace Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Flows
 {
     internal class AcquireTokenByDeviceCodeHandler : AcquireTokenHandlerBase
     {
-        private readonly DeviceCodeResult deviceCodeResult;
+        private readonly DeviceCodeResult _deviceCodeResult;
+        private readonly CancellationToken _cancellationToken;
```

Comment:
Wouldn't it be better place in teh base?

Reply:
Yup, ADAL. In MSAL, the cancellation token is part of the method call itself as it should be, but i don't want to re-wire all of RunAsync() across the actions for adal.